### ISSUE TITLE
numAnalogInputChannels cannot be zero

### ISSUE THREAD

**Issue by sensestage** [sensestage/supercollider#22]
scsynth -u 57110 -z 16

**Comment by sensestage** [comment]
From this discussion: http://forum.bela.io/d/53-supercollider-on-bela/3

**Comment by sensestage** [comment]
Ok, the error actually happens in `Bela/core/RTAudio.cpp` around line 180; it should allow for numAnalogInputChannels = 0 if useAnalog = 0;

**Comment by giuliomoro** [comment]
We currently do this by disabling analog, it is 

```
settings->useAnalog
```

or, from Bela command line:

```
   --use-analog [-N] val:              Set whether to use ADC/DAC analog (default: 1)
```

**Comment by sensestage** [comment]
Fixed with 2138d774ede9d40a507707d1f150b4cdbd4b922c
